src/pages/[platform]/build-a-backend/existing-resources/cdk/index.mdx

```diff
@@ -46,15 +46,15 @@ Before you begin, you will need:
 <InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
 
 * An AWS account: If you don't already have an account, follow the [Setting Up Your AWS Environment](https://aws.amazon.com/getting-started/guides/setup-environment/) tutorial for a quick overview.
-* The Amplify CLI [installed](https://docs.amplify.aws/cli/start/install/) and configured.
```

Comment:
To test this one, visit https://docs.amplify.aws/cli/start/install/ and verify that it redirects to `/[platform]/tools/cli/start/set-up-cli/`